documentation/docs/components/audio.md

```diff
@@ -1,5 +1,7 @@
 # Audio
 
+Audio refers to any form of sound or sound-related content that can be played, recorded, processed or interacted within the app. Audio can include various types of sound such as music, spoken dialogue, sound effects and other auditory elements.
```

Comment:
I think this describes the word audio, but not the audio player component in the app(?)

Reply:
A lot better, thanks. I don't think it has the ability to adjust volume at the moment, but the rest is fine.